src/main/java/duke/command/CommandExecutor.java

```diff
@@ -0,0 +1,117 @@
+package duke.command;
+
+import duke.datasaver.DataManager;
+import duke.parser.Parser;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import duke.exception.EmptyListException;
+import duke.exception.InvalidCommandFormatException;
+
+public class CommandExecutor {
+
+    private final TaskList taskList;
+
+    /** Constructor */
```

Comment:
remove redundant comments. As a reader your comments are not adding any value to make your code more understandable. your comments should describe the what and why of your logic